test/runAnalysis/runAnalysis.py

```diff
@@ -56,6 +56,15 @@ def getFilterSnippet(name,args,doApply=False,year=2017):
 				       )
 for path_name, path in process.paths.iteritems():
 	getattr(process,path_name).insert(2,process.DYGenMassFilter)'''
+        ADDFilter1700 = '''
+process.DYGenMassFilter = cms.EDFilter('QScaleSelector',
+                                       src = cms.InputTag('generator'),
+                                       min_mass = cms.double(1700),
+                                       max_mass = cms.double(7000)
+                                       )
+for path_name, path in process.paths.iteritems():
+        getattr(process,path_name).insert(2,process.DYGenMassFilter)'''
+
```

Comment:
Why is this necessary? The lower cut of 1700 is baked into the sample and you don't have to apply it here. And why do you an upper cut of 7000?

Reply:
You are right! I just checked the files with and without this cut, they are actually the same!